src/lib/component/audio_post.dart

```diff
@@ -2,36 +2,75 @@ import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:flutter/material.dart';
 import 'package:provider/provider.dart';
 import 'package:yamabico/feature/posts/presentation/index_screen.dart';
+import 'package:yamabico/feature/user/presentation/user_detail_screen.dart';
 
 class AudioPost extends StatelessWidget {
   final int index;
+  final bool isVisibleAvatar;
   final AudioData audioData;
 
-  const AudioPost({super.key, required this.index, required this.audioData});
+  const AudioPost({
+    super.key,
+    required this.index,
+    this.isVisibleAvatar = true,
+    required this.audioData,
+  });
+
+  // TODO: userDetailへの遷移はルーティングを通して遷移させる
+  void _navigateToUserDetail(BuildContext context) {
+    Navigator.push(
+      context,
+      MaterialPageRoute(
+        builder: (context) => UserDetailScreen(
+          user: audioData.user,
+          userPosts: [audioData],
+        ),
+      ),
+    );
+  }
 
   @override
   Widget build(BuildContext context) {
     final itemState = Provider.of<ItemState>(context);
+
     return Container(
-        padding: const EdgeInsets.all(8.0),
-        child: Row(
-          children: [
-            Avatar(url: audioData.user.avatarUrl),
-            Content(
-                title: audioData.title,
-                name: audioData.user.name,
-                time: audioData.playTime,
-                count: audioData.count,
-                date: audioData.date),
-            IconButton(
-                icon: itemState.isPlaying(index)
-                    ? const Icon(Icons.pause_circle_filled)
-                    : const Icon(Icons.play_circle_filled),
-                color: const Color.fromRGBO(124, 122, 122, 1.0),
-                iconSize: 40,
-                onPressed: () => itemState.onPressedPlayButton(index))
-          ],
-        ));
+      padding: const EdgeInsets.all(8.0),
+      child: Row(
+        children: [
+          isVisibleAvatar
+              ? GestureDetector(
+                  onTap: () => _navigateToUserDetail(context),
+                  child: Padding(
+                    padding: const EdgeInsets.only(right: 13.0),
+                    child: Avatar(
+                      url: audioData.user.avatarUrl,
+                      width: 70,
+                      height: 60,
+                    ),
+                  ),
+                )
+              : const SizedBox(
+                  width: 0,
+                  height: 0,
+                ),
```

Comment:
[nits]
`width`とか`height`なくてもいけるみたいでした！


```suggestion
    const SizedBox()
```

Reply:
ありがとうございます！修正しました！